pandas/core/generic.py

```diff
@@ -231,6 +231,10 @@ def _init_mgr(self, mgr, axes=None, dtype=None, copy=False):
     def attrs(self) -> Dict[Optional[Hashable], Any]:
         """
         Dictionary of global attributes on this object.
+
+        .. warning::
+
+           attrs is experimental and may change without warning.
```

Comment:
Thanks! Can you also add this in the reference docs (where you added it in https://github.com/pandas-dev/pandas/pull/29062/files)